src/main/java/com/kansas/TaigaAPI/controller/MilestoneController.java

```diff
@@ -135,4 +136,9 @@ public JsonNode getGivenSprintData(int sprintId, JsonNode allSprintData){
         return null;
     }
 
+    @GetMapping("/{projectId}/getTotalPoints")
```

Comment:
Should it not be projectSlug?

Reply:
I sent the url format the frontend is expecting, please check on slack

Reply:
addressed